packages/manager/tools/webpack-config/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "@ovh-ux/manager-webpack-config",
+  "version": "3.0.8",
+  "description": "OVH manager shared webpack configuration",
+  "keywords": [
+    "configuration",
+    "control-panel",
+    "manager",
+    "ovh",
+    "shared",
+    "webpack"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh-ux/manager.git",
+    "directory": "packages/manager/tools/webpack-config"
+  },
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
+  "files": [
+    "dist"
+  ],
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "tsc",
+    "dev": "tsc -w",
+    "lint": "eslint --quiet --fix src",
+    "prepare": "yarn run build",
+    "test": "eslint --quiet src",
+    "version": "npx conventional-changelog-cli -p angular -i CHANGELOG.md -s && git add CHANGELOG.md",
+    "postversion": "git push && git push --tags"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.2.2",
+    "@babel/generator": "^7.2.2",
+    "@babel/parser": "^7.2.3",
+    "@babel/plugin-syntax-dynamic-import": "^7.2.0",
+    "@babel/preset-env": "^7.2.3",
+    "@commitlint/cli": "^7.5.2",
+    "@commitlint/config-angular": "^7.5.0",
+    "@ovh-ux/component-rollup-config": "^5.1.0",
+    "@ovh-ux/manager-webpack-dev-server": "^2.1.0",
```

Comment:
Pay attention. An issue has been opened to provide the latest version of `@ovh-ux/manager-webpack-dev-server`.

Here you are importing the latest version that hasn't be tested with the actual configuration (https://github.com/ovh-ux/manager/pull/986/files?file-filters%5B%5D=.json&file-filters%5B%5D=.lock&file-filters%5B%5D=.md#diff-4722f9be2756722c66d3df3f9a4f947bR3)

ref: https://github.com/ovh-ux/manager-webpack-config/issues/74